src/modules/utils/ISingleVoteGovernor.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.8.0;
 
 import {IAuthorizer} from "src/modules/authorizer/IAuthorizer.sol";
 
-interface ISingleVoteGovernor is IAuthorizer {
```

Comment:
It can be set up as only address with the owner role, forcing all owner-limited actions to go through governance

Reply:
So this just makes the contract a kind of a custom implementation of a multisig. But then why maintain such a contract when solutions like Safe can give you the same or even better functionality, they are well maintained and audited, and will reduce maintenance need from you?

If that's the case I'd recommend to delete this contract and just use Gnosis Safe or a similar solution.

Reply:
Good point, although it's more akin to a simplified Compound GovernorBravo than a multisig. Your main point stands, though. It may be a better option to wrap an external Governor implementation that we don't need to maintain, rather than use our own. Moving discussion to #376